<?xml version="1.0" ?>
<launch>
    <arg name="x" default="8"/>
    <arg name="y" default="8"/>
    <arg name="pause" default="false"/>
    <arg name="eval" default="false"/>

    <arg name="pause_arg" default="no_pause" unless="$(arg pause)"/>
    <arg name="pause_arg" value="pause" if="$(arg pause)"/>

    <param name="/map/configuration_map" type="string" value="$(find maru2_map)/saved_maps/configuration_space/room.pgm" />
    <param name="/map/free_space_map" type="string" value="$(find maru2_map)/saved_maps/free_space/room.pgm" />
    <param name="/map/voxel_size" type="double" value="0.05" />

    <node name="wavefront" pkg="local_motion" type="wavefront.py" output="screen"
          args="$(arg x) $(arg y) $(arg pause_arg)" cwd="node" required="true"/>

    <group if="$(arg eval)">
        <node name="wavefront_eval" pkg="local_motion" type="eval_length.py" output="screen" cwd="node"/>
    </group>
</launch>
